scriptmodules/libretrocores/lr-mgba.sh

```diff
@@ -40,11 +40,12 @@ function install_lr-mgba() {
 }
 
 function configure_lr-mgba() {
-    mkRomDir "gba"
-    ensureSystemretroconfig "gba"
-
     local def=1
     isPlatform "armv6" && def=0
-    addEmulator $def "$md_id" "gba" "$md_inst/mgba_libretro.so"
-    addSystem "gba"
+    for system in gb gbc gba; do
+        mkRomDir "$system"
+        ensureSystemretroconfig "$system"
+        addEmulator "$def" "$md_id" "$system" "$md_inst/mgba_libretro.so"
```

Comment:
I'm not certain if @joolswills wants this patch included at all, but if so, perhaps he can chime in on whether the previous defaults should be preserved, or if all three systems should use ```lr-mgba``` as set by this patch (I think not).